shared/utils.sh

```diff
@@ -32,19 +32,19 @@ function get_metadata() {
     local key=$1
     local metadata_ip='169.254.169.254'
 
-    if [ ! -z "$IMDSV2" ]; then
+    if [ ! -z "$(get_config_value .imdsv2)" ]; then
         local token=$(curl -s -X PUT "http://$metadata_ip/latest/api/token" -H "X-aws-ec2-metadata-token-ttl-seconds: 60")
         local token_wrapper='-H "X-aws-ec2-metadata-token: $token"'
```

Comment:
I'm testing this locally and it doesn't expand out properly when using single quotes.

```
> token_wrapper='-H "X-aws-ec2-metadata-token: $token"'
> echo $token_wrapper
-H "X-aws-ec2-metadata-token: $token"
```

Changing it to this works
```
> token_wrapper='-H "X-aws-ec2-metadata-token: '"$token"'"'
> echo $token_wrapper
-H "X-aws-ec2-metadata-token: <snip>"
```

Reply:
Nevermind, I see that `eval` fixes it...

Reply:
Ah!!! I didnt see this PR till just now. I ran into the same issue of $token not expanding properly. My fix is slightly different: #62